### PR TITLE
Added React Native trigger words.

### PR DESCRIPTION
**Pull request by pjhampton**
## Description of new Instant Answer, or changes

The new React-Native Fathead is badly under triggering. I suspect this is because there are no trigger words. I have added them here quickly instead of creating an issue and waiting for someone to do it.

## People to notify

@moollaza 
@Hestben 

## Testing & Review
To be completed by Language Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - One topic for the Search Space Language (Java, Python, Scala, Ruby, etc.)
    - One topic from: Reference, Help, Libraries, Tools
        - Documentation Fatheads are considered "Reference"
    - Description, Data source, and 2+ example queries
    - Perl Module (e.g. "DDG::Fathead::PerlDoc" -- we only need a name, not an actual file)
    - Source Name (for "More at <source_name>" link)
    - Source Domain (must contain http:// or https:// -- can be the same as Data Source)
    - Source Info (used as Subtitle for each Article -- usually matches the IA Name)
    - 'Skip Abstract' is checked off
    - Source ID (ping @moollaza to assign one, once Fathead is ready for Beta deploy)

**Code**
- [ ] Uniformly indented, well commented
- [ ] Fetch.sh and Parse.sh run without errors
- [ ] Output contains no blank lines, or multi-line entries
- [ ] Fathead Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/react_native

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by Hestben** [comment]
Oh, yeah. I was meaning to ask about trigger words when creating the IA, but it vanished from my mind during the work. Is the internals adding all combinations of using "react-native", "React-native", "React-Native" as well?

**Comment by pjhampton** [comment]
Hey @Hestben 

The query is normalized in a way that `-` are dropped and casing is typically ignored. For example:

react component, triggers: https://duckduckgo.com/?q=react+component&t=h_&ia=about
react comPonent, triggers: https://duckduckgo.com/?q=react+comPonent&t=h_&ia=about
etc

**Comment by Hestben** [comment]
@pjhampton So, the dash is dropped internally? I.e. the trigger word "react-native" implies "react native" as well?

**Comment by pjhampton** [comment]
I _think_ so. It's hard to tell because the query parsing is closed source and I can't see it. Understanding comes from word-of-mouth. 

I ran some tests and in some cases the `-` is dropped, such as https://duckduckgo.com/?q=pep+427&t=h_&ia=about, there was only one case where this wasn't so.

File: lib/fathead/react_native/trigger_words.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+react mobile
+react-native
```
Comment (pjhampton) [inline lib/fathead/react_native/trigger_words.txt]: I spoke with @moollaza. If you just put `react-native` and `react native` in for now. We're not entirely sure what the case is with trigger words. 👍

**Comment by moollaza** [comment]
@pjhampton Thanks! This LGTM :+1:

**Pull request merged**